sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestObjectSerializer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    /// <summary>
+    /// Custom serializer to pass to the DigitalTwins client.
```

Comment:
What does it do? How do we know it was used?

Reply:
I does not do anything special, just uses the JsonObjectSerializer internally. I chose to do that as we just want to test passing of a custom serializer as options and not any logic of how a serializer is implemented.

Reply:
I tested with a breakpoint but I will add a field to ensure that we know it was used in the test. It cannot be accessed directly so I'll have to add something to the serializer.